tools/config/index.ts

```diff
@@ -278,7 +278,11 @@ manageDependencies(repository, options);
 manageTsConfigs(repository, options);
 
 // collect and generate dependencies.json file.
+if (PUBLISHING_VERSION) {
```

Comment:
Would there be a way to make this more deliberate, like, for example passing a param to the script?